### DISCUSSION TITLE
How was the version 11 conflict fixed with netstandard 2.0?

### DISCUSSION THREAD

**Discussion by TalesReig** [LuckyPennySoftware/MediatR#812] [Q&A]
Why does MediatR 11.1.0 not give an error with netstandard 2.0 and 11.0.0 gives an error ?
What was changed in the project that allows this compatibility of 11.1.0 with netstandard 2.0?

**Comment by jbogard** [comment]
MediatR 11.1.0 still targets `netstandard2.0`. I'm not sure why you'd not get an error.

However, the MediatR.Contracts library did change to target `netstandard2.0`. Maybe that's what you're seeing?

**Reply by TalesReig** [reply]
So only 11.1.0 is compatible with netstandard2.0? 
Because 11.0.0 didn't work with netstandard2.0.
I discovered this when I was trying to update MediatR in a project and found this error, in the case of 11.0.0 it only works if used with netstandard2.1, and 11.1.0 with 2.0.
If you want, I can simulate the error here in my test project.

**Reply by jbogard** [reply]
From [NuGet](https://www.nuget.org/packages/MediatR):

### MediatR
|Version|Target Framework|
|-|-|
|`11.1.0`|`netstandard2.1`|
|`11.0.0`|`netstandard2.1`|
|`10.0.1`|`netstandard2.1`|
|`9.0.0`|`netstandard2.1`, `netstandard2.0`, `net461`|

### MediatR.Contracts
|Version|Target Framework|
|-|-|
|`1.0.1`|`netstandard2.0`, `net461`|
|`1.0.0`|`netstandard2.1`|

### MediatR version dependency on MediatR.Contracts
|MediatR Version|MediatR.Contracts Version|
|-|-|
|`11.1.0`|`1.0.1`|
|`11.0.0`|`1.0.1`|
|`10.0.1`|`1.0.1`|
|`10.0.0`|`1.0.0`|
|`9.0.0`|`n/a`|

As you can see, absolutely nothing changes with respect to dependencies or versions from 11.0.0 to 11.1.0, nor from 10.0.1 to 11.0.0. The last change in dependencies happened from 10.0.0 to 10.0.1, when I changed the Contracts assembly's target frameworks to be lower.
